### DISCUSSION TITLE
404 error - File not found.

### DISCUSSION THREAD

**Discussion by jkindla** [community/community#38632] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello!

I have been working on this website, and everything clears when I view it in live server (using VS code), but when I push it to GitHub and try to make the website go live with GitHub pages I am getting a 404 error stating the File is not found.

Upon researching the issue, I've double checked the following:

-I have my home screen titled 'index.html' and it is located in the root folder.
-Although I am using a custom domain name 'victoryaross.com', the problem still persists when I remove the domain name and use the URL 'https://jkindla.github.io/victoryaross.com/'
-A curious note: I was able to get it up and running briefly as victoryaross.com, but I needed to edit as I forgot a meta tag and when I pushed using GitHub Desktop, the problem continued.
-I've cleared my cache and waited 48hrs.
-I've created a duplicate repository and the same issue occurs
-I've tried adding '/index.html' to the end of the Web address and the same error

Here are some speculations:

-I am going to check if my GitHub desktop app is up to date
-I am an amateur coder at best, and although I have a lot of editing to do on this current iteration, I know there is a lot of unnecessary lines in the files.

If anyone is able to offer any guidance or advice on this issue it would be much appreciated! The repository link is https://github.com/jkindla/victoryaross.com.git
[https://github.com/jkindla/victoryaross.com.git](https://github.com/community/community/discussions/url)

Best,

Joshua

**Comment by mdAshik922** [comment]
If it doesn't work for you here, you can try deploying your project folder to [Netlife](https://www.netlify.com/)

**Reply by Pandeycoder** [reply]
A "404 error - File not found" indicates the requested resource isn't available. Check URL spelling, file existence, permissions, server configuration, and debug tools for resolution.

**Reply by Rosensjs195** [reply]
sounds simple enough. I have my one repository and I just share an .apk download, and some read me . txt but sending it to someone to determine if they could get access to the repository by way of link, they had encountered the 404 error. My project is private. I think they would have to be included as a member of git hub.

**Comment by DSDmark** [comment, marked as answer]
Hi @jkindla,

here is your site you want to deploy like this:- 
https://dsdmark.github.io/victoryaross.com/

For deploying your site first go your **repo** and open **settings** and on **Code and automation** there is **Environment,** click on that and delete you all **Environments** first.

![Firefox_Screenshot_2022-11-09T03-51-49 679Z](https://user-images.githubusercontent.com/85744816/200734333-020c9cae-5de0-4951-bd3a-71616f25a448.png)

and also open you **repo** and delete your that folder

![Firefox_Screenshot_2022-11-09T04-03-54 725Z](https://user-images.githubusercontent.com/85744816/200735772-7f3db84c-974f-42bc-8216-ee8a71bce43c.png)

After that on **Code and automation** there is **page** click on it and in **Build and deployment** there is source option click on that and select **github-action** and after that you can see there is two option appear.

- static site
- jakyll 


![Firefox_Screenshot_2022-11-09T03-58-54 125Z](https://user-images.githubusercontent.com/85744816/200735302-a2dff9ad-a4b1-4af9-9128-7ba51b12329f.png)

> note:- if you see your page like that then first **upbulish** your page first by click on **three dot** then you see **static site and jakyll site**.

![Firefox_Screenshot_2022-11-09T04-15-43 865Z](https://user-images.githubusercontent.com/85744816/200737416-c1007cc6-5c6b-4070-b995-e5df9d38fc77.png)

Select Jekyll and commit your **action**.

![Firefox_Screenshot_2022-11-09T03-59-24 675Z](https://user-images.githubusercontent.com/85744816/200735332-1a7cc78f-2ea5-44a9-9844-1828b0c60f30.png)

Now you can see your deployment page.

**Reply by DSDmark** [reply]
and also fix your css.

**Reply by jkindla** [reply]
Thank you @DSDmark I followed your steps carefully and it worked! Thank you for the very detailed response and for taking the time to help out. I really appreciate it!

**Reply by Zeph180** [reply]
@DSDmark  
Can you please help me on how to remove custom domain from my github pages

**Reply by DSDmark** [reply]
Hi @Zeph180,

just go to your repository and open settings and inside the settings there is `page` section, click on it and remove your domain. Like this.

![image](https://user-images.githubusercontent.com/85744816/208418613-88dd4ab1-f2ad-4a73-8bf8-a937027e7ce1.png)

And also if you have `CNAME` file in your repository. Then, also delete it.

**Reply by DSDmark** [reply]
And If you need help. Then, don't hesitate to ask @Zeph180.

**Reply by raghuvartandon15** [reply]
> Hi @jkindla,
> 
> here is your site you want to deploy like this:- https://dsdmark.github.io/victoryaross.com/
> 
> For deploying your site first go your **repo** and open **settings** and on **Code and automation** there is **Environment,** click on that and delete you all **Environments** first.
> 
> ![Firefox_Screenshot_2022-11-09T03-51-49 679Z](https://user-images.githubusercontent.com/85744816/200734333-020c9cae-5de0-4951-bd3a-71616f25a448.png)
> 
> and also open you **repo** and delete your that folder
> 
> ![Firefox_Screenshot_2022-11-09T04-03-54 725Z](https://user-images.githubusercontent.com/85744816/200735772-7f3db84c-974f-42bc-8216-ee8a71bce43c.png)
> 
> After that on **Code and automation** there is **page** click on it and in **Build and deployment** there is source option click on that and select **github-action** and after that you can see there is two option appear.
> 
> * static site
> * jakyll
> 
> ![Firefox_Screenshot_2022-11-09T03-58-54 125Z](https://user-images.githubusercontent.com/85744816/200735302-a2dff9ad-a4b1-4af9-9128-7ba51b12329f.png)
> 
> > note:- if you see your page like that then first **upbulish** your page first by click on **three dot** then you see **static site and jakyll site**.
> 
> ![Firefox_Screenshot_2022-11-09T04-15-43 865Z](https://user-images.githubusercontent.com/85744816/200737416-c1007cc6-5c6b-4070-b995-e5df9d38fc77.png)
> 
> Select Jekyll and commit your **action**.
> 
> ![Firefox_Screenshot_2022-11-09T03-59-24 675Z](https://user-images.githubusercontent.com/85744816/200735332-1a7cc78f-2ea5-44a9-9844-1828b0c60f30.png)
> 
> Now you can see your deployment page.

hi @DSDmark
my website works on GitHub pages but as soon as I use a custom domain name, I can only see a blank white screen with "[Error] Failed to load resource: the server responded with a status of 404 () (index-alFhsSWy.js, line 0)" in the console. could you help me

**Reply by DSDmark** [reply]
Hi @raghuvartandon15,
if you are not seeing your deployment page after adding custom domain name then please check your **A record** which should correct redirecting to your GitHub page URL. You can add a screenshot of your config, which you did.

**Comment by methmiranasinghe** [comment]
You can try redeploying it using GitHub Pages or you can use the Heroku platform which is free to deploy your application or otherwise you can use Microsoft Azure as well

**Comment by airtower-luna** [comment]
> -A curious note: I was able to get it up and running briefly as victoryaross.com, but I needed to edit as I forgot a meta tag and when I pushed using GitHub Desktop, the problem continued.

Here's your problem: https://github.com/jkindla/victoryaross.com/commit/1964012343342d32b07296bb2e6ccc9fbbee74b7 You deleted the configuration file that sets the custom domain.

**Reply by kuxav** [reply]
Error in /docs : no such directory. Well... use GitHub Pages classic.

**Reply by airtower-luna** [reply]
Right, that too. Adjusting the source directory will be needed as well.

**Reply by jkindla** [reply]
I was fortunately able to deploy the site with success using 'GitHub Actions' as described by @DSDmark. But I will be sure to double check for these items as well with my continued projects. Thank you both @kuxav & @airtower-luna for taking a look and for your willingness to help!

**Comment by kuxav** [comment]
Wait... You use GitHub Pages classic. Well, use / (read: root) but you chose /docs for build. Set /docs to root at Settings >> Pages >> set /docs to / (root). If no trigger, can change a file or someting.

@jkindla , maybe can help about source directory.

**Comment by Zeph180** [comment]
Man the  remove button does not activat e
[image: image.png]

**Reply by DSDmark** [reply]
Hi @Zeph180 ,

Can you **specific more** about the details? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Reply by Zeph180** [reply]
![Screenshot from 2022-12-19 15-03-37](https://user-images.githubusercontent.com/58359456/208428233-24566534-308e-4176-9759-b921afde0b99.png)
here is a screenshot

**Reply by DSDmark** [reply]
@Zeph180, Did you try to remove input field?. Did you able to select URL to input field like this?

![image](https://user-images.githubusercontent.com/85744816/208434011-8b8d4628-7f78-4091-bfd5-46bc68aba616.png)

**Reply by Zeph180** [reply]
@DSDmark 
Unfortunately no. I typed the whole domain my self because there was no option for me to select

**Reply by DSDmark** [reply]
So, did you try to `save` it Then tries to remove?

**Reply by Zeph180** [reply]
@DSDmark 
Yeah i surely did but still the remove button does not activate
Can I invite to the repo so you can help me from with in

**Comment by Zeph180** [comment]
Yeah sure here is a screenshot

**Comment by DSDmark** [comment]
Hi @Zeph180,

  -  Make sure that you are using a compatible web browser and that it is up to date.

   - Try clearing your browser cache and cookies.

  -  Try using a different device or internet connection to access GitHub.

  -  If you are using a VPN or proxy, try disabling it and accessing GitHub directly.

  -  If you are using a script blocker or ad blocker, try disabling it and accessing GitHub again.

   - If you are still unable to click on the "Remove" button, you may want to contact GitHub support for further assistance. [**Contact Support**](https://support.github.com/request) button.

**Reply by Zeph180** [reply]
Alright thanks for you time @DSDmark

**Comment by LTanyaMari** [comment]
Unable to view websites.  Please view pic below.   

![image.png](https://github.zendesk.com/attachments/token/J2a2Nrcozw8twlyHkjhSQ1Mhi/?name=image.png)

I will accept how to instructions on how to fix the 404 error message.

**Comment by Tyjay00** [comment]
Please help as I'm trying to remove the Namecheap domain from my Github but my sites are still being redirected to the Namecheap domain

![namecheap](https://github.com/community/community/assets/125516735/6464c3a5-5d36-47fb-8e24-48b458d963ba)

**Comment by kuxav** [comment]
Check CNAME file in root repo project,
Configure site url or site domain make sure back to domain to
username.github.io, then
Build next this

Pada Kam, 9 Mei 2024 07.44, Tyrone Joel ***@***.***> menulis:
